packages/core/template_project/app/src/main/AndroidManifest.xml

```diff
@@ -124,12 +124,22 @@
             <% if (shareTarget) { %>
                 <intent-filter>
                     <action android:name="android.intent.action.SEND" />
-                    <action android:name="android.intent.action.SEND_MULTIPLE" />
+                    <% 
+                       // SEND_MULTIPLE is added when handling images, videos, etc.  
+                       if (shareTarget.params.files) { %>
+                        <action android:name="android.intent.action.SEND_MULTIPLE" />
```

Comment:
Is there any reason not to combine this block with the one below?

Reply:
Just keeping a nice order in the `AndroidManifest.xml`. FWIW, I'm leaving this as a draft as I'm thinking I'd rather move this logic somewhere else and just have 2 for loops (one of actions and another for mime types)

Reply:
Ah fair enough - in which case I'll also point out how seeing you use C-style comments in the template section of an XML file also confuses me.

Reply:
I honestly find this whole section hard to read... (that's why I want to move the logic)

Reply:
Rewrote this bit - still need to test better, so leaving this as a draft for now.